.github/workflows/upstream_ci.yml

```diff
@@ -127,25 +127,18 @@ jobs:
           use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
 
       - name: Setup environment
-        shell: bash -l {0}
+        shell: bash -el {0}
```

Comment:
This makes it fail if there is no `dev/macos/setup.sh`?

Reply:
Yes. This is an oversight we had at `setup-miniconda` and I am slowly adopting the new `shell` across projects. [See issue here](https://github.com/conda-incubator/setup-miniconda/issues/227).